includes/wikia/Wikia.php

```diff
@@ -2264,12 +2264,10 @@ static function canEditInterfaceWhitelist (&$title, &$wgUser, $action, &$result)
 
 		// In this NS, editinterface applies only to white listed pages and staff users
 		if (in_array($title->getDBKey(), $wgEditInterfaceWhitelist) || in_array('staff', $wgUser->getGroups())) {
```

Comment:
Yes, it makes staff users going through the whitelist and edit all pages if they have rights (which they have).